pom.xml

```diff
@@ -854,7 +854,7 @@
         <carbon.kernel.registry.imp.pkg.version.range>[1.0.1, 2.0.0)</carbon.kernel.registry.imp.pkg.version.range>
 
         <!-- Carbon Identity Framework version -->
-        <carbon.identity.framework.version>5.15.58</carbon.identity.framework.version>
+        <carbon.identity.framework.version>5.16.82</carbon.identity.framework.version>
```

Comment:
Need to update the framework version after a new version is released with the changes of https://github.com/wso2/carbon-identity-framework/pull/2781